src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -369,6 +382,11 @@ internal override void Validate()
             {
                 throw new InvalidOperationException(MsalErrorMessage.InvalidRedirectUriReceived(Config.RedirectUri));
             }
+
+            if (Config.IsBrokerEnabled && Config.MultiCloudSupport)
+            {
+                throw new NotSupportedException(MsalErrorMessage.MultiCloudSupportUnavailable);

```

Comment:
nit: pls make sure we have a tracking item for the broker work.

Reply:
I did create an issue in github to track this. Will add it as a comment.

Reply:
I thought the mobile platforms already had support for this? Or is there something preventing us from enabling this on the broker from our side?

Reply:
We split this work to first implement it for web and then work for broker to make it simpler. And there is a team waiting for this work on the web.